doc/guide.rst

```diff
@@ -4,19 +4,15 @@ Getting started
 Before installing or launching the `Universum`, please make sure your system meets the following
 :doc:`prerequisites`.
 
-The main script (`universum.py`) is used for performing all CI-related actions.
-If using raw sources, launch Universum via running this script with :doc:`parameters <args>`::
+The main module (`universum`) is used for performing all CI-related actions.
+If using raw sources, launch Universum from project root via running main module with :doc:`parameters <args>`::
 
-    $ ./universum.py --help
-
-
-If using a module installed via PyPi, use created ``universum`` command from any suitable directory::
-
-    $ universum --help
+    $ python3.7 -m universum --help
 
+If using a module installed via PyPi, use the same command from any suitable directory.
```

Comment:
I think it is enough to say "installed module", because it could be installed from other sources, not only from PyPi